salt/salt.spec

```diff
@@ -829,6 +830,9 @@ than serially.
 Summary:        The client component for Saltstack
 Group:          System/Management
 Requires:       %{name} = %{version}-%{release}
+%if 0%{?suse_version} > 1500 || 0%{?sle_version} > 1500
```

Comment:
I think `sle_version` has more numbers. I think it needs to be 150000.

We have other packages which use this:

`%if 0%{?sle_version} >= 120000`

Reply:
Sigh ... indeed again. Changed to `%if 0%{?suse_version} > 1500 || 0%{?sle_version} > 150000`